applications/stepswitch/src/stepswitch_app.erl

```diff
@@ -19,6 +19,10 @@
 -spec start(application:start_type(), any()) -> startapp_ret().
 start(_StartType, _StartArgs) ->
     _ = declare_exchanges(),
+    _ = kazoo_bindings:bind(
```

Comment:
When you drop lines for lists of things, the convention is to have the comma precede the item. So

```erlang
kazoo_bindings:bind(kapps_maintenance:binding('refresh_stepswitch')
                   ,'stepswitch_maintenance'
                   ,'refresh'
                   ),
```

Would be more in line with the project's style. There is a style guide as a Google doc that I want to convert into a markdown in the repo for reference. I'll try to get that into the repo this week.